kafka-connect-s3/src/test/java/io/confluent/connect/s3/TestWithMockedS3.java

```diff
@@ -76,7 +77,9 @@
   @Override
   public void setUp() throws Exception {
     super.setUp();
-    port = url.substring(url.lastIndexOf(":") + 1);
+    if (StringUtils.isBlank(port)) {
```

Comment:
Should we add a comment here telling if an explicit port is already set by child testcase do not overwrite it?